community/lvm2/build

```diff
@@ -14,9 +15,8 @@ sed -i 's/fmt -1/tr " " "\\n"/' \
 # swap to using 'install -d' instead which works.
 export MKDIR_P="install -d"
 
-# Build system requires bash
-# https://www.redhat.com/archives/linux-lvm/2020-January/msg00004.html
-export CONFIG_SHELL=/bin/bash
+# Bashism banished.
+export CONFIG_SHELL=/bin/sh
```

Comment:
Can this not be fully removed now?

Reply:
I'm preparing for the next version of LVM2 which would hardcode bash entirely.

Reply:
Alright. :+1: